software/SchemaExamples/schemaexamples.py

```diff
@@ -141,25 +140,19 @@ def hasValidId(self):
 
 
     def serialize(self):
-        buff = []
-        termnames = ""
-        first = True
-        idd = "#%s" % self.keyvalue
-        if "-temp-" in idd:
-            idd = ""
-        for t in self.terms:
-            if first:
-                first = False
-            else:
-                termnames += ", "
-            termnames += t
-
-        buff.append("TYPES: %s %s\n" % (idd,termnames))
-        buff.append("PRE-MARKUP:\n%s" % self.getHtml())
-        buff.append("MICRODATA:\n%s" % self.getMicrodata())
-        buff.append("RDFA:\n%s" % self.getRdfa())
-        buff.append("JSON:\n%s" % self.getJsonld())
-        return "\n".join(buff)
+       termnames = ', '.join(self.terms)
+       idd = "#%s" % self.keyvalue
+       if "-temp-" in idd:
+           idd = ""
+
+       sections = [
+         "TYPES: %s %s\n" % (idd, termnames),
+         "PRE-MARKUP:\n%s" % self.getHtml(),
+         "MICRODATA:\n%s" % self.getMicrodata(),
+         "RDFA:\n%s" % self.getRdfa(),
+         "JSON:\n%s" % self.getJsonld(),
```

Comment:
All entries except the first ends with substitution on a new line, i.e.: `{HEADER}:\n%s", self.get{}(),`
Could it simply be:

```python
"PRE-MARKUP:",
self.getHtml(),
"MICRODATA:",
self.getMicrodata(),
# ...
```

Reply:
Done